src/lib/rest/rest.cpp

```diff
@@ -922,14 +924,19 @@ static ApiVersion apiVersionGet(const char* path)
     return V2;
   }
 
-  // Different from v2, v1 is case-insensitive (see case/2057 test)
```

Comment:
Yes, the change was necessary when I had the ngsi9 and ngsi10 stats there.
However, to not repeat code, I might add an enum to ConnectionInfo and then I will need the separation.